data/lua/wml/items.lua

```diff
@@ -48,6 +52,23 @@ function wesnoth.interface.remove_item(x, y, name)
 	end
 end
 
+---@class item_info
+---@field x integer
+---@field y integer
+---@field image string
+---@field halo string
+---@field team_name string
+---@field filter_team WML
+---@field visible_in_fog boolean
+---@field redraw boolean
+---@field name string
+---@field z_order integer
+---@field variables WMLTable
+
+---Get items on a given hex
+---@param x integer
+---@param y integer
```

Comment:
I haven't tested, but can these be ranges instead?

Reply:
They're an index to a `location_set`, so no.